### DISCUSSION TITLE
Adding branch protections

### DISCUSSION THREAD

**Discussion by MgenGlder** [dearborn-coding-club/website-base-backend#9] [Announcements]
I've recently added branch protections to this repo- so now you can only merge into `main` with a pull request, and you must have at least 1 review before a pull request can be merged.

cc/ @dXu23

**Comment by imaricodes** [comment]
Good idea. I need to update my shared repos as such.

**Reply by MgenGlder** [reply]
Yeah it's pretty rad- I'd also like to take a look at setting up CI so test failures/successes are shown on pull request. I suppose we first have to write some tests though 😅 . One step at a time.
